### PR TITLE
chore: add changelog

### PR DESCRIPTION
**Pull request by jennijuju**
add changelog for v1.18.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,37 +1,65 @@
...
+  - This is a transitional state to enabling far more efficient and dynamic storage markets on Filecoin network in the future.
+  - ⭐️ First Fungible Token Contract - Datacap Actor, on Filecoin!  ([fungible token standard](https://github.com/filecoin-project/FIPs/blob/master/FRCs/frc-0046.md), [token contract library](https://github.com/helix-onchain/filecoin/tree/5455f4f831e0f3f20005a9a789623d7ad6dada15/frc46_token)).
+  - For storage deal participants (clients and storage providers):
+    - `PublishStorageDeals`/`ProveCommit(Aggregate)`/`ProveReplicaUpdates` message that includes verified deals will see a gas usage increase, more details can be found [here](https://github.com/filecoin-project/FIPs/blob/385f069b3b146c5fef4fdc1109a0e2f35f399e48/FIPS/fip-0045.md?plain=1#L784)
+    -  `Term` is introduced for defining how long the DataCap is assigned to a piece of data. Anyone who cares about that piece of data may extend the _term_, which incentives SPs to store the data longer on the network without a new deal/resealing.
+    - There is no  more diluted verified deal QAP due to deal/sector space time.
```
Comment (geoff-vball) [inline CHANGELOG.md:24]: Probably not this is for new sectors only

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,37 +1,65 @@
...
+The Shark upgrade introduces the following FIPs, delivered in [actors v9](https://github.com/filecoin-project/builtin-actors/releases/tag/v9.0.3):
 - [FIP0029 Beneficiary Address for Storage Providers](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0029.md): step towards better lending market for SP
 - [FIP0034 Fix PreCommit Deposit Independent of Sector Content](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0034.md): resolves a significant weakening of Filecoin PoRep’s security guarantees
-  - ❗Pre-commit deposit will be calculated as the 20-day projection of expected reward earned by a sector with a sector quality of 10 (i.e. full of verified deals), regardless of sector content. Leave the initial pledge value, due when the sector is proven, unchanged.
+  - ❗Pre-commit deposit will be calculated as the 20-day projection of expected reward earned by a sector with **a sector quality of 10 (i.e. full of verified deals)**, regardless of sector content. Leave the initial plede value, due when the sector is proven, **unchanged**.
```
Comment (geoff-vball) [inline CHANGELOG.md:15]:
```suggestion
  - ❗Pre-commit deposit will be calculated as the 20-day projection of expected reward earned by a sector with **a sector quality of 10 (i.e. full of verified deals)**, regardless of sector content. Leaves the initial pledge value, due when the sector is proven, **unchanged**.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,37 +1,65 @@
...
 We are planning to switch [the snapshot service listed in lotus docs](https://lotus.filecoin.io/lotus/manage/chain-management/#lightweight-snapshot) to the new Lightweight Filecoin Chain Snapshots Service by EOY, and deprecate public support of the current snapshots production. We recommend all users to test and switch the new service ASAP, and if you run into any issue, please report them [here](https://github.com/filecoin-project/filecoin-chain-archiver/discussions/new?category=feedback) and the team would be happy to support you! For the main differences between the old & the new service, checkout the FAQ section [here](https://www.notion.so/pl-strflt/Lightweight-Filecoin-Chain-Snapshots-17e4c386f35c44548f5863afb7b5e024)
 
 ## Migration 
 
-(TBC)
+We are expecting a heavier than normal state migration for this upgrade due to the amount of the state changes introduced.
+All node operators, including storage providers, should be aware that two pre-migration is being scheduled. The first pre-migration will begin at 2022-11-30T12:00:00Z (120 minutes before the real upgrade), the second migration will begin at 2022-11-30T13:45:00Z (7.5 minutes before the real upgrade).
```
Comment (geoff-vball) [inline CHANGELOG.md:36]:
```suggestion
All node operators, including storage providers, should be aware that two pre-migrations are being scheduled. The first pre-migration will begin at 2022-11-30T12:00:00Z (120 minutes before the real upgrade), the second pre-migration will begin at 2022-11-30T13:45:00Z (7.5 minutes before the real upgrade). 
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,37 +1,65 @@
...
 
 ## Migration 
 
-(TBC)
+We are expecting a heavier than normal state migration for this upgrade due to the amount of the state changes introduced.
+All node operators, including storage providers, should be aware that two pre-migration is being scheduled. The first pre-migration will begin at 2022-11-30T12:00:00Z (120 minutes before the real upgrade), the second migration will begin at 2022-11-30T13:45:00Z (7.5 minutes before the real upgrade). 
+The first pre-migration will take up to 1.5hr, depending on the amount of the historical state in the node blockstore and the hardware specs the node is running on. During this time, expect slower block validation times, increased CPU and memory usage, and longer delays for API queries. 
+We recommend node operators(who hasn't enbabled splistore `universal` mode) that do not care about historical chain states, to prune the chain blockstore by syncing from a snapshot 1-2 days before the upgrade.
```
Comment (geoff-vball) [inline CHANGELOG.md:38]:
```suggestion
We recommend node operators (who haven't enbabled splistore `universal` mode) that do not care about historical chain states, to prune the chain blockstore by syncing from a snapshot 1-2 days before the upgrade.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -70,21 +97,36 @@ We are planning to switch [the snapshot service listed in lotus docs](https://lo
 - Delete lotus-pond (#9352) ([filecoin-project/lotus#9352](https://github.com/filecoin-project/lotus/pull/9352))
 - build: set version to v1.18.0-dev
 
+## lotus-market EOL notice 
+
+As mentioned in [lotus v1.17.0 release notes](https://github.com/filecoin-project/lotus/releases/tag/v1.17.0), markets related features, enhancements and fixes is now lower priority for Lotus. We recommend our users to migrate to other deal making focused softwares, like [boost](https://boost.filecoin.io/) as soon as possible. That being said, the lotus maintainers will be:
```
Comment (geoff-vball) [inline CHANGELOG.md:102]:
```suggestion
As mentioned in [lotus v1.17.0 release notes](https://github.com/filecoin-project/lotus/releases/tag/v1.17.0), markets related features, enhancements and fixes is now lower priority for Lotus. We recommend our users to migrate to other deal making focused software, like [boost](https://boost.filecoin.io/) as soon as possible. That being said, the lotus maintainers will be:
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -70,21 +97,36 @@ We are planning to switch [the snapshot service listed in lotus docs](https://lo
...
 |-------------|---------|---------|---------------|
-| Geoff Stuart | 51 | +8677/-19320 | 401 |
-| Aayush Rajasekaran | 5 | +1452/-166 | 34 |
-| Łukasz Magiera | 5 | +429/-135 | 45 |
-| Aayush | 19 | +281/-157 | 72 |
-| Shrenuj Bansal | 3 | +176/-61 | 10 |
-| Jennifer Wang | 7 | +19/-18 | 15 |
-| simlecode | 1 | +5/-5 | 4 |
+| @geoff-vball | 73 | +14533/-19712 | 509 |
+| @arajasek | 16 | +2230/-303 | 49 |
```
Comment (geoff-vball) [inline CHANGELOG.md:113]: Two arajaseks and THREE jennijujus

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,37 +1,65 @@
...
-# 1.18.0 / 2022-11-14
+# 1.18.0 / 2022-11-15
 
 > ⚠️ **Please note that from Lotus v1.17.2&^ will require a Go-version of v1.18.1&^**
 
 This is the final release of the upcoming MANDATORY release of Lotus that introduces [Filecoin network v17,
 codenamed the Shark upgrade](https://github.com/filecoin-project/community/discussions/74?sort=top#discussioncomment-3825422). Shark upgrade delivers a wave of protocol refinements that will allow for useful smart contracts to be written using the FVM (eg. programmable markets, lending contracts, etc.).
 
-A full changelog will be published later tonight.
+**The Filecoin mainnet is scheduled to upgrade to nv17 at epoch 2383680, on Nov 30th on 2022-11-30T14:00:00Z. All node operators, including storage providers, must upgrade to this release (or a later release) before that time. Storage providers must update their daemons, miners, market and worker(s)/boost.**
```
Comment (geoff-vball) [inline CHANGELOG.md:10]:
```suggestion
**The Filecoin mainnet is scheduled to upgrade to nv17 at epoch 2383680, on Nov 30th on 2022-11-30T14:00:00Z. All node operators, including storage providers, must upgrade to this release before that time. Storage providers must update their daemons, miners, market and worker(s)/boost.**
```

File: CHANGELOG.md (hunk 1 above)
Comment (jennijuju) [reply CHANGELOG.md:24]: good call will add

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,37 +1,65 @@
...
+The Shark upgrade introduces the following FIPs, delivered in [actors v9](https://github.com/filecoin-project/builtin-actors/releases/tag/v9.0.3):
 - [FIP0029 Beneficiary Address for Storage Providers](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0029.md): step towards better lending market for SP
 - [FIP0034 Fix PreCommit Deposit Independent of Sector Content](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0034.md): resolves a significant weakening of Filecoin PoRep’s security guarantees
-  - ❗Pre-commit deposit will be calculated as the 20-day projection of expected reward earned by a sector with a sector quality of 10 (i.e. full of verified deals), regardless of sector content. Leave the initial pledge value, due when the sector is proven, unchanged.
+  - ❗Pre-commit deposit will be calculated as the 20-day projection of expected reward earned by a sector with **a sector quality of 10 (i.e. full of verified deals)**, regardless of sector content. Leaves the initial pledge value, due when the sector is proven, **unchanged**.
```
Comment (arajasek) [inline CHANGELOG.md:15]: ❗Pre-commit deposit will be calculated as the 20-day projection of expected reward earned by a sector with **a sector quality of 10 (i.e. full of verified deals)**, regardless of sector content. The Initial Pledge value, due when the sector is proven, is left **unchanged**.

**Review by arajasek — APPROVED** [review]
One suggestion, but LGTM.

**Pull request merged**